package.json

```diff
@@ -25,7 +25,7 @@
     "postcheckout": "node bin/post-checkout.js",
     "postmerge": "node bin/post-merge.js",
     "preinstall": "node bin/pre-install.js",
-    "prepush": "yarn test-all",
+    "prepush": "yarn test-all && yarn test-alex",
```

Comment:
I think this should go to `test-all` instead ? It should be as simple as adding `test-alex` at the end of `test-all`.

Reply:
yes that makes much more sense.

Reply:
I don't see the update to this yet, did you forget to add the change ?

Reply:
yes, completely forgot